docs/release-notes/release-notes-0.16.0.md

```diff
@@ -276,6 +282,13 @@ data.
   link is falling behind, which is now fixed by [retrying the enable
   request](https://github.com/lightningnetwork/lnd/pull/7157). 
 
+* [Add the possibility to define a change address
+  for FundPsbt in the cli](https://github.com/lightningnetwork/lnd/pull/7348)
+  using 'change-address' flag. If no address type is specified for
+  default/imported accounts, P2WKH will be used. No custom address type should
+  be provided for custom accounts as we will always use the coin selection key
+  scope to generate the change address.
```

Comment:
I don't think lncli needs a separate entry in the release notes. At the other mention, you can optionally add a sentence saying that the functionality is also available in lncli.